spec/calendar.html

```diff
@@ -630,9 +630,7 @@ <h1>Temporal.Calendar ( _id_ )</h1>
         1. Set _id_ to ? ToString(_id_).
         1. If ! IsBuiltinCalendar(_id_) is *false*, then
           1. Throw a *RangeError* exception.
-        1. Let _calendar_ be ? OrdinaryCreateFromConstructor(NewTarget, *"%Temporal.Calendar.prototype%"*, « [[InitializedTemporalCalendar]], [[Identifier]] »).
-        1. Set _calendar_.[[Identifier]] to _id_.
-        1. Return _calendar_.
+        1. Return ? CreateTemporalCalendar(_id_, NewTarget).
```

Comment:
I share the confusion re: jordan's prior comment - what is NewTarget here? It seems it must be undefined.

Reply:
Jordan's comment was about a copy-paste error that I believe I fixed — [NewTarget is a spec-text notation that's relevant when evaluating a constructor with the [[Construct]] internal method](https://tc39.es/ecma262/#sec-built-in-function-objects-construct-argumentslist-newtarget) while _newTarget_ is a variable. I had accidentally copy-pasted the latter into this context where I should've used the former, but I believe this is correct now.

Reply:
that said, it seems kind of nicer to do the NewTarget defaulting in the constructor directly, and let the AO always take a constructor.

Reply:
The constructor already throws if NewTarget is undefined, so I think this is covered?